libs/design/src/core/openable/openable.directive.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  Directive,
+  EventEmitter,
+  HostBinding,
+  Input,
+  Output,
+} from '@angular/core';
+
+import { DaffOpenable } from './openable';
+
+@Directive({
+  selector: '[daffOpenable]',
+  standalone: true,
+})
+
```

Comment:
Remove the whitespace.

Reply:
Disagree. There should be a empty new line?

<img width="480" alt="Screen Shot 2024-08-12 at 9 14 39 AM" src="https://github.com/user-attachments/assets/dd3ff8e5-c2b7-4e80-9ae6-5a3df0be37cc">